src/Elastic.CommonSchema.BenchmarkDotNetExporter/Elastic.CommonSchema.BenchmarkDotNetExporter.csproj

```diff
@@ -3,9 +3,8 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;netstandard2.1;net461</TargetFrameworks>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
-
-    <Title>ECS BencharkDotNet exporter</Title>
-    <Description>Exports BenchmarkDotNet benchmarks to ECS format</Description>
+    <Title>Elastic Common Schema (ECS) BencharkDotNet exporter</Title>
+    <Description>Exports BenchmarkDotNet benchmarks to Elastic Common Schema (ECS) format</Description>
```

Comment:
does it also index them into Elasticsearch, or just export them in that format?

Reply:
Indexes too, via some helpers, will update